### PR TITLE
Add FAQ tab content

### PR DESCRIPTION
**Pull request by mhuth**
Having problems pulling the content dynamically from https://access.redhat.com/hydra/rest/drupal/articles/4602981 so hardcoding it for the time being until I sort it out (or give up and run away).

### CODE REVIEW

**Comment by mhuth** [comment]
Screenshot from 2020-09-09 21-32-50

File: package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-intl": "5.8.0",
+    "react-markdown": "^4.3.1",
```
Comment (AllenBW) [inline package.json:59]: gottal lock this version down 😬 🤗

File: src/Components/Tabs/FAQ.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import messages from '../../Messages';
+import React, { useState, useEffect } from 'react';
+import ReactMarkdown from 'react-markdown/with-html';
+import { useIntl } from 'react-intl';
+
+const FAQ = () => {
+    const [content, setContent] = useState();
+    const intl = useIntl();
+
+    useEffect(() => {
```
Comment (AllenBW) [inline src/Components/Tabs/FAQ.js:14]: lets comment out the block if it doesn't work yet, no sense in pounding n api for no reason 💯

File: src/Components/Tabs/FAQ.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    width: 70%;
+    background-color: var(--pf-global--BackgroundColor--100);
+    margin-top: var(--pf-global--spacer--xl) !important;
+    padding: var(--pf-global--spacer--xl);
+}
+
+p {
+    margin-bottom: var(--pf-global--spacer--md);
+}
+ul {
```
Comment (AllenBW) [inline src/Components/Tabs/FAQ.scss:11]: needs ah space aboveeee

File: src/Components/Tabs/FAQ.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+.ins-c-card {
```
Comment (AllenBW) [inline src/Components/Tabs/FAQ.scss:1]: i think this is n existing pf class name, might want to rename to something else, more specific to the app

BUT we should probably use things that are similar to whats in cardcollection.scss, maybe bubble that styling up to app.scss, make it common across?

File: src/Messages.js [hunk 5]
```diff
@@ -421,5 +421,10 @@ export default defineMessages({
...
...s not operating at maximum speed due to faulty cable, network interface card, switchport, SFP, etc.</li> <li>Unsupported kernel version on Intel Purley Platform with Intel Skylake CPU</li> <li>Kdump Does Not Work Due To XEN/AWS\'s Limitation</li> </ul> </blockquote> <p><strong>Q: Does Insights support Red Hat Enterprise Linux running on IBM Power Systems and IBM Z systems?</strong><br /> <strong>A:</strong>Yes, Insights works on these hardware platforms and provides analysis of general RHEL operations on these platforms.</p> <p><strong>Q: Where can I get additional information about Red Hat Insights?</strong><br /> <strong>A</strong>: There are a range of internal and external resources on Red Hat Insights:<br /> - <a href="http://redhat.com/insights" target="_blank">Red Hat Insights product web page</a><br /> - <a href="https://access.redhat.com/products/red-hat-insights" target="_blank">Red Hat Insights Customer Portal page</a><br /> - <a href="https://www.youtube.com/user/RedHatVideos/search?query=insights" target="_blank">Video Channel: Red Hat Insights</a><br /> - <a href="https://www.redhat.com/en/blog/channel/red-hat-insights" target="_blank">Blog: Red Hat Insights</a></p> '
```
Comment (AllenBW) [inline src/Messages.js:428]: lawl, what ah long line HAHAHA HAVE FUN TRANSLATING THAT ONE 🌞

**Review by AllenBW — CHANGES_REQUESTED** [review]
just a few things! but m real excited forrrrrrr this... makes it look sao official :godmode:

**Review by AllenBW — APPROVED** [review]
delicious! yeah m down with this 🙇 🤗 👍 nicely done @mhuth

**Pull request merged**